src/common/log/log.ts

```diff
@@ -50,7 +50,7 @@ export module Log {
      * them to them.
      */
     export function logInternalMessage(logLevel: LogLevel, message: string) {
-        if (LogHelper.logLevel >= logLevel) {
+        if (LogHelper.internalLogLevel >= logLevel) {
```

Comment:
I find this to be extraneous. How about just using a higher non-default logLevel as the bar for internal messages?